### DISCUSSION TITLE
Remove/deprecate JCenter resolver

### DISCUSSION THREAD

**Discussion by Philippus** [sbt/sbt#7761] [Ideas]
I think the JCenter resolver (jcenter.bintray.com) now redirects to Maven. Should it be removed/deprecated?

**Comment by eed3si9n** [comment]
https://jfrog.com/blog/jcenter-sunset/

**Comment by Philippus** [comment]
https://github.com/sbt/sbt/issues/7806

**Comment by eed3si9n** [comment]
Closing this now.
